shell/platform/darwin/macos/BUILD.gn

```diff
@@ -66,14 +66,13 @@ source_set("flutter_framework_source") {
     "framework/Source/FlutterView.mm",
     "framework/Source/FlutterViewController.mm",
     "framework/Source/FlutterViewController_Internal.h",
-    "framework/Source/MacOSSwitchableGLContext.h",
-    "framework/Source/MacOSSwitchableGLContext.mm",
+    "framework/Source/MacOSGLContextSwitch.h",
+    "framework/Source/MacOSGLContextSwitch.mm",
   ]
 
   sources += _flutter_framework_headers
 
   deps = [
-    "//flutter/flow:flow",
     "//flutter/fml:fml",
```

Comment:
I think we don't want to depend on `fml` as well.

Reply:
Right. Removed.

Reply:
FML is actually okay; we're going to need it for a11y support, so I've looked into it already, and it's generally very base-level, with minimal dependencies.

Reply:
In this instance it was only used for single FML_DCHECK call, which I've replaced with NSAssert. I can put it back if FML_DCHECK is better.